Hibernate envers for db versioning (#203)
jpa-buddy/documentation: installation-guide/installation-guide.md

    <td class="tg-0lax"><a href="https://mvnrepository.com/artifact/com.blazebit/blaze-persistence-integration-spring-data-base" target="_blank" rel="noopener noreferrer">Blazebit Persistence Integration Spring Data Base</a></td>
    <td class="tg-0lax">- Create Spring Data JPA repository for Blaze Persistence Entity View.</td>
  </tr>
  <tr>
    <td class="tg-0lax"><a href="https://mvnrepository.com/artifact/org.hibernate/hibernate-envers" target="_blank" rel="noopener noreferrer">Hibernate Envers</a></td>
    <td class="tg-0lax">
    - Generate database migration scripts for audit tables.<br>
    - Create Spring Data JPA revision repositories.<br>

>     - Mark entities and its fields that need to be audited via JPA Designer. <br>

Reviewer: maybe "Annotate entities and their fields for auditing via JPA Designer."?

>     - Mark entities and its fields that need to be audited via JPA Designer. <br>

Author: ```suggestion
    - Annotate entities and their fields for auditing via JPA Designer. <br>
```
